docs/security-and-authentication.rst

```diff
@@ -34,3 +34,15 @@ local Runhouse config (:code:`~/.rh/config.yaml`), or in Python:
 
     import runhouse as rh
     rh.configs.disable_data_collection()
+
+
+Disabling Cluster Telemetry Recording
```

Comment:
Decided to call it Cluster Telemetry Recording after taking a look at some Ray [documentation](https://docs.ray.io/en/latest/_modules/ray/serve/handle.html#). Don't have strong opinion about that though.